Core/MMKV.h

```diff
@@ -288,7 +288,7 @@ class MMKV {
     bool set(const std::vector<std::string> &vector, MMKVKey_t key);
     bool set(const std::vector<std::string> &vector, MMKVKey_t key, uint32_t expireDuration);
 
-    bool getString(MMKVKey_t key, std::string &result);
+    bool getString(MMKVKey_t key, std::string &result, bool useReference = false);
```

Comment:
The naming of `useReference` is ambiguous because the argument `result` is already passed by references. Consider a different name.